serving/samples/jib-app-java/sample.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: jib-sample-app
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      build:
+        serviceAccountName: riff-build
```

Comment:
The auth instructions use build-bot, is this correct?

Reply:
oops, will remove.